src/System.Runtime.Serialization.Primitives/src/Configurations.props

```diff
@@ -4,7 +4,7 @@
     <BuildConfigurations>
       netcore50aot-Windows_NT;
       uap101aot-Windows_NT;
-      net463-Windows_NT;
+      netfx-Windows_NT;
```

Comment:
If you want to block certain verticals from building that's a different story.

I don't think it makes sense to configure every project to be OSGroup specific just so you don't get a binary when you specify a config that *might* be invalid (I don't agree, btw, what about mono?).

Even if we did this for all projects targeting netfx, you'd still get binaries dropped for netstandard configs that don't specify an OSGroup when building for vertical netfx-Unix.

IMHO without any enforcement, you're just forcing folks to type more.